model/project_parser_test.go

```diff
@@ -1188,3 +1188,11 @@ tasks:
+	proj := &Project{}
+	assert.NoError(LoadProjectInto([]byte(confStr), "mci", proj))
+	assert.NotNil(proj.FindTaskGroup("lint-group"))
```

Comment:
IIRC this was part of debugging, but this test doesn't actually prove anything about the bug in this PR. If so, I think we can remove this test.